src/cecs277TermProject/AmenityFoodBar.java

```diff
@@ -0,0 +1,19 @@
+package cecs277TermProject;
+
+public class AmenityFoodBar  extends AmenitiesDecorator {

```

Comment:
We should think of these classes are room classes as it's extended from `Room.java`, so I think we should name them with `Room` as a prefix or suffix instead of `Amenity`. For example, we can name it `RoomWithFoodBar`.